docs/example-relationship-definition.yml

```diff
@@ -0,0 +1,54 @@
+# a list of relationships defined in a file under the entity in which the telemetry is sourced
```

Comment:
In the real `relationships.yml` files you are adding a query on top to see the data, that seems useful, should we suggest that people add it?

Reply:
ya! nice suggestion

Reply:
+1 although I would probably use natural language to explan the rule (as the user may not necessarily understand the query approach)

Reply:
done!

Reply:
thanks!